src/docs/issue-platform.mdx

```diff
@@ -118,6 +120,21 @@ There's a minimum set of fields which should be included:
 Caveats
 - Attachments will not work. If you have a need for this please contact the issues team.
 
+### Payload Type
+<Link to="https://github.com/getsentry/sentry/blob/ccfb37212102945df5f35e8f55fe38891b70036b/src/sentry/issues/producer.py#L23">PayloadType</Link>
+
+Issue platform supports two types of payloads:
+- `OCCURRENCE`
+- `STATUS_CHANGE`
+
+### Staus Change
+The `status_change` message format is as follows:
```

Comment:
Let's add a sentence here to explain what this message is used for, since I don't see that explained anywhere else.

Reply:
I have 

> If `payload_type` is set to `STATUS_CHANGE` in `produce_occurrence_to_kafka`, `status_change` will update the status of an issue.